retype.yml

```diff
@@ -1,7 +1,6 @@
 input: .
 output: .retype
-url: https://geoffreymcgill.github.io/
-base: artblocks-docs
+url: https://geoffreymcgill.github.io/artblocks-docs/
```

Comment:
We'll also want to change this to https://docs.artblocks.io and I can update our DNS.

Reply:
> We'll also want to change this to https://docs.artblocks.io/ and I can update our DNS.

Done. Super simple change:

https://github.com/ArtBlocks/artblocks-docs/pull/38/files#diff-51e3f34ae21758348d19efce1c16d17b146daf28e7a9da5ba2950677185717eeR3

I also revised the page structure in the Retype project so it exactly replicates the the existing docs.artblocks.io URL paths. This means the Retype project generates a drop-in replacement for the GitBook generated website. Any existing inbound links to docs.artblocks.io would continue to resolve correctly. There _should_ be no 404's with any inbound links from the wild.

Reply:
Woohoo, thank you very much! :)